app/packages/embeddings/src/Embeddings.tsx

```diff
@@ -96,13 +98,15 @@ export default function Embeddings({ containerHeight, dimensions }) {
                 <PlotOption to={() => resetZoom()} title={"Reset zoom (Esc)"}>
                   <CenterFocusWeak />
                 </PlotOption>
-                <PlotOption
-                  style={{ opacity: dragMode !== "lasso" ? 0.5 : 1 }}
-                  to={() => setDragMode("lasso")}
-                  title={"Select (s)"}
-                >
-                  <HighlightAlt />
-                </PlotOption>
+                <Box data-cy="embeddings-plot-option-lasso" display="flex">
```

Comment:
Out of curiosity, why was `Box` added? Was `flex` missing?

Reply:
to do with `data-cy` mainly. The alternative is to pass cy property down to the Link component. I'll change the code to pass a cy property down to Link instead. That way we can reuse that for other `PlotOption`s